components/Context/UrlButton.tsx

```diff
@@ -0,0 +1,51 @@
+// UrlButton.tsx
+
+import { Button } from "./Button";
+import React, { useContext, FC } from "react";
+import Link from "next/link";
+import selectedPDFsContext from "@/context/selected-context";
```

Comment:
i dont think u need a separate component for selectedContext, u can just add that code into this file